addons/mail/static/src/web/chatter.scss

```diff
@@ -50,3 +50,7 @@
 .btn.o-mail-Chatter-follow:hover {
     color: $black !important;
 }
+
+.o-mail-Chatter-content {
+    background-color: $o-webclient-background-color;
+}
```

Comment:
There's an equivalent rule for `o-mail-Chatter-header`. I think you can define this rule on `.o-mail-Chatter` so we don't have to repeat it on header and content

Reply:
Ho great it's event better.